src/source/Signaling/LwsApiCalls.c

```diff
@@ -1400,19 +1415,34 @@ STATUS connectSignalingChannelLws(PSignalingClient pSignalingClient, UINT64 time
     BOOL locked = FALSE;
     SERVICE_CALL_RESULT callResult = SERVICE_CALL_RESULT_NOT_SET;
     UINT64 timeout;
+    SignalingResult_t retSignal;
+    SignalingRequest_t signalRequest;
+    SignalingChannelEndpoint_t wssEndpoint;
+    ConnectWssEndpointRequestInfo_t connectWssEndpointRequestInfo;
 
     CHK(pSignalingClient != NULL, STATUS_NULL_ARG);
     CHK(pSignalingClient->channelEndpointWss[0] != '\0', STATUS_INTERNAL_ERROR);
+    CHK(pSignalingClient->channelDescription.channelArn[0] != '\0', STATUS_INVALID_OPERATION);
 
-    // Prepare the json params for the call
+    // Prepare URL buffer
+    signalRequest.pUrl = &url[0];
+    signalRequest.urlLength = MAX_URI_CHAR_LEN + 1;
+    // Prepare body buffer
+    signalRequest.pBody = NULL;
+    signalRequest.bodyLength = 0;
+    // Prepare WSS endpoint
+    wssEndpoint.pEndpoint = pSignalingClient->channelEndpointWss;
+    wssEndpoint.endpointLength = STRLEN(pSignalingClient->channelEndpointWss);
+    MEMSET(&connectWssEndpointRequestInfo, 0, sizeof(ConnectWssEndpointRequestInfo_t));
```

Comment:
SIZEOF. Also, the name of the type is not adhering to the naming conventions

Reply:
As mentioned above, this is provided in Signaling library.

Reply:
Updated